packages/solid/src/types/form.ts

```diff
@@ -82,3 +82,9 @@ export type FormState<TFieldValues extends FieldValues> = {
   invalid: boolean;
   response: Response;
 };
+
+/* 
+  Extracts the field values from the form state.
```

Comment:
Please adjust the formatting of the comment. Example:

```ts
/**
 * Extracts the field values from the form state.
 */
```

Reply:
Done, also added a bit better comment to follow what is already established in other comments